common/djangoapps/student/views/dashboard.py

```diff
@@ -531,6 +533,21 @@ def _credit_statuses(user, course_enrollments):
     return statuses
 
 
+def show_load_all_courses_link(load_all_courses, user, courses_count):
+    """
+    By default dashboard will show limited courses based on the course limit
+    set in configuration.
+
+    A link is provided at the bottom to load all the courses if there are any courses.
+    """
+
+    total_enrollments = CourseEnrollment.enrollments_for_user(user).count()
+    if not load_all_courses and courses_count < total_enrollments:
+        return True
+
+    return False
```

Comment:
Small nit (and this is totally optional): Sometimes it's helpful to do early returns for logic like this, even if it's a little redundant. For instance, you could do:

```python
if load_all_courses:
    return False

return courses_count < total_enrollments
```

That last line could return `True` or `False`, but it can read easier for people because their brain can short circuit the comparisons. "If they're already loading all the courses, of course we shouldn't show the load all courses link."

That's not as big a deal in a case like this where the number of items is relatively small, but it becomes very helpful as the boolean logic gets bigger.